src/Sulu/Component/Persistence/EventSubscriber/ORM/UserBlameSubscriber.php

```diff
@@ -123,6 +123,8 @@ public function onFlush(OnFlushEventArgs $event)
 
     private function handleUserBlame(OnFlushEventArgs $event, UserInterface $user, bool $insertions)
     {
+        // TODO: calling getEntityManager on the event is deprecated, but access to the UnitOfWork is not
+        // guaranteed for an ObjectManager. So we need to get an EntityManager somehow different.
         $manager = $event->getEntityManager();
         $unitOfWork = $manager->getUnitOfWork();
```

Comment:
Think the suggestion of @stof make sense we just check if its an entity manager event if not early return.

```suggestion
        $manager = $event->getObjectManager();
        if (!$manager instanceof EntityManagerInterface {
            return;
        }
        
        $unitOfWork = $manager->getUnitOfWork();
```

Reply:
Thank you, I have changed the PR according to your suggestion.